include/rofl/ofdpa/rofl_ofdpa_fm_driver.hpp

```diff
@@ -144,8 +148,13 @@ class rofl_ofdpa_fm_driver final {
                                                        uint32_t lport_no);
   cofflowmod remove_bridging_unicast_overlay_all_tunnel(uint8_t ofp_version,
                                                         uint32_t tunnel_id);
+  cofflowmod add_bridging_multicast_vlan(uint8_t ofp_version, uint32_t port_no,
+                                         uint16_t vid, const cmacaddr &mac);
```

Comment:
we really should have a unified naming for mac addresses, it randomly varies between mac, dmac, dst